server/src/utils/utils.js

```diff
@@ -52,3 +52,13 @@ export function isNumericInputValid(values) {
     return true;
   }
 }
+
+export function isEmailInputValid(values) {
+  const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
I think this regex could be improved. For example, the email sahar@gmail.c is valid according to this regular expression, but the single character extension is not valid.

Reply:
It is just to avoid to overload the client, but for sure can be improved

Reply:
no it is not valid. Try it in regex101.com